src/60-tls/internal.cc

```diff
@@ -147,16 +147,15 @@ void tls::initialize() {
     log_v2::tls()->error(
         "TLS: could not load TLS Diffie-Hellman parameters: {}",
         gnutls_strerror(ret));
-    throw(exceptions::msg()
-          << "TLS: could not load TLS Diffie-Hellman parameters: "
-          << gnutls_strerror(ret));
+    throw msg_fmt("TLS: could not load TLS Diffie-Hellman parameters: {}",
+                  gnutls_strerror(ret));
   }
   ret = gnutls_dh_params_import_pkcs3(dh_params, &dhp, GNUTLS_X509_FMT_PEM);
   if (ret != GNUTLS_E_SUCCESS) {
-    log_v2::tls()->error(
-        "TLS: could not import PKCS #3 parameters: ", gnutls_strerror(ret));
-    throw(exceptions::msg() << "TLS: could not import PKCS #3 parameters: "
-                            << gnutls_strerror(ret));
+    log_v2::tls()->error("TLS: could not import PKCS #3 parameters: ",
```

Comment:
There is an error here. You are not the guilty :laughing:  But please, could you fix it please?
"TLS: could not import PKCS #3 parameters: {}"